test/parallel/test-eventtarget-whatwg-once.js

```diff
@@ -0,0 +1,93 @@
+// Manually converted from https://github.com/web-platform-tests/wpt/blob/master/dom/events/AddEventListenerOptions-once.html in order to define the `document` ourselves
+
+// Flags: --expose-internals
+'use strict';
+
+// eslint-disable-next-line no-unused-vars
+const common = require('../common');
+
+const {
+  Event,
+  EventTarget,
+} = require('internal/event_target');
+
+const {
+  strictEqual
+} = require('assert');
+
+{
+  const document = new EventTarget();
+  let invoked_once = false;
+  let invoked_normal = false;
+  function handler_once() {
+    invoked_once = true;
+  }
+
+  function handler_normal() {
+    invoked_normal = true;
+  }
+
+  document.addEventListener('test', handler_once, { once: true });
```

Comment:
We have the `common.mustCall()` utility that can be used to verify that a function is called a given number of times. 
```suggestion
  document.addEventListener('test', common.mustCall(), { once: true });
```

Reply:
Ahh i see - okay I'll make this change throughout the test file

Reply:
For functions that are called more than once, or that you need to remove later, you can do...

```js
// handler_normal is a function that must be called twice
const handler_normal = common.mustCall(2);
```

Reply:
I converted the first test. Have to spend more time with the others later